assets/js/script.js

```diff
@@ -0,0 +1,548 @@
+var $ball1 = $('#ball1')
+var $ball2 = $('#ball2')
+var $body = $('body')
+var goalLevel = 1 // to assign goal Id's for each play field
+var gravity = 50 // gravity multiplier
+var platformNo = 1 // to assign platform Id's
+var gLock = false // toggle for gravity
+var allowance = 1 // to accomodate for gravity stopping distance, i.e. hover value
+var jumpLimit = true // switch for jump
+var start1 = [] // indicates ball1 starting position, index position increases with level
+var start2 = []
+var currentLevel
+var lose = new Audio('assets/sounds/lose.wav')
+var loseCheck = false
+var jumpSound = new Audio('assets/sounds/jump.wav')
+var muteSwitch = true // mute switch
+var muteBeep = new Audio('assets/sounds/beep.wav')
+var switchPress = new Audio('assets/sounds/switchpress.wav')
+
+$(function () {
+  $('body').on('keydown', startClick)
+  function startClick (e) { if (e.keyCode === 32) $('.start').click() }
+
+  $('.start').on('click', () => {
+    $('.start').remove()
+    $('h2').remove()
+    $('.half').removeClass('fade') // .fade greys out background when play (.play) is off
+    $('.half').addClass('play')
+    $body.on('keydown', ballMove)
+    function ballMove (e) {
+      var unit = 20 // 20
+      if (e.key === 'd' || e.key === 'D' || e.key === 'ArrowRight') {
+        if (ball1Goal() === false && (borderCheck1Right()) && (wallCheck1Right()) && (!$('.fade').length)) ball1Hori(unit) // if ball1 doesn't exceed right border
+        if (ball2Goal() === false && (borderCheck2Left()) && (wallCheck2Left()) && (!$('.fade').length)) ball2Hori(unit) // opposite side for ball2
+      }
+      if (e.key === 'a' || e.key === 'A' || e.key === 'ArrowLeft') {
+        if (ball1Goal() === false && (borderCheck1Left()) && (wallCheck1Left()) && (!$('.fade').length)) ball1Hori(-unit)
+        if (ball2Goal() === false && (borderCheck2Right()) && (wallCheck2Right()) && (!$('.fade').length)) ball2Hori(-unit)
+      }
+      if (e.key === 'w' || e.key === 'W' || e.key === 'ArrowUp') {
+        if (jumpLimit) {
+          if (!ball1Goal()) ball1Jump()
```

Comment:
Why can't just there be one function that checks goal for both ball?

Reply:
This is to let both balls be independent from each other, when a single ball goes into the goal, it will be frozen, while the other ball can still move/jump.